gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -106,10 +106,12 @@ enum
   PROP_INPUTTYPE,
   PROP_INPUTNAME,
   PROP_INPUTLAYOUT,
+  PROP_INPUTRANKS,
   PROP_OUTPUT,
   PROP_OUTPUTTYPE,
   PROP_OUTPUTNAME,
   PROP_OUTPUTLAYOUT,
+  PROP_OUTPUTRANKS,
```

Comment:
need to remove these.

Reply:
Do you mean that 'Output RANKs' is not necessary?

Reply:
If this is for get-property, it's ok.